mongodbatlas/fw_resource_mongodbatlas_encryption_at_rest_test.go

```diff
@@ -116,10 +117,11 @@ func TestAccAdvRSEncryptionAtRest_basicAWS(t *testing.T) {
 			SecretAccessKey:     os.Getenv("AWS_SECRET_ACCESS_KEY_UPDATED"),
 			CustomerMasterKeyID: os.Getenv("AWS_CUSTOMER_MASTER_KEY_ID_UPDATED"),
 			Region:              os.Getenv("AWS_REGION_UPDATED"),
+			RoleID:              os.Getenv("AWS_ROLE_ID"),
 		}
 	)
 
-	resource.ParallelTest(t, resource.TestCase{
```

Comment:
curious about this change, was it for a particular reason?

Reply:
Yes, these tests currently do not run in CI and are only run locally. When running all tests locally together if we try to run all tests at once that results in inconsistencies because all of them currently use the same projectID to set/update/delete the same resource.

Reply:
Tried this and we can't create separate projects for each test in the corresponding TF config because that messes with the checkDestroy functionality in test.

Reply:
Thanks for looking into this and providing context.